docs/src/pages/css-in-js/advanced/advanced.md

```diff
@@ -196,16 +199,16 @@ export default function MyComponent() {
 }
 ```
 
-The hook call order or the class name concatenation orders **don't matter**.
+The hook call order and the class name concatenation order **don't matter**.
 
 ### insertionPoint
 
-JSS [provides a mechanism](https://github.com/cssinjs/jss/blob/master/docs/setup.md#specify-the-dom-insertion-point) to gain more control on this situation.
-By adjusting the placement of the `insertionPoint` within your HTML head you can [control the order](https://cssinjs.org/jss-api#attach-style-sheets-in-a-specific-order) that the CSS rules are applied to your components.
+JSS [provides a mechanism](https://github.com/cssinjs/jss/blob/master/docs/setup.md#specify-the-dom-insertion-point) to control this situation.
+By adding an `insertionPoint` within the HTML you can [control the order](https://cssinjs.org/jss-api#attach-style-sheets-in-a-specific-order) that the CSS rules are applied to your components.
 
 #### HTML comment
 
-The simplest approach is to add an HTML comment that determines where JSS will inject the styles:
+The simplest approach is to add an HTML comment to the `<head>` that determines where JSS will inject the styles:
```

Comment:
```suggestion
The simplest approach is to add a HTML comment to the `<head>` that determines where JSS will inject the styles:
```

Reply:
'an` is correct, since 'aitch' begins with a vowel.